frontend/www/js/omegaup/components/arena/Arena.vue

```diff
@@ -62,18 +67,28 @@
 
 <script lang="ts">
 import { Vue, Component, Prop, Emit } from 'vue-property-decorator';
+import { types } from '../../api_types';
 import T from '../../lang';
 import { Tab } from '../problem/Details.vue';
 
 @Component
 export default class Arena extends Vue {
   @Prop({ default: false }) shouldShowRuns!: boolean;
+  @Prop({ default: false }) isAdmin!: boolean;
+  @Prop({ default: () => [] }) clarifications!: types.Clarification[];
   @Prop() contestTitle!: string;
   @Prop() activeTab!: string;
   @Prop() backgroundClass!: string;
 
   T = T;
   selectedTab = this.activeTab;
+  clarificationsHasBeenRead = true;
+
+  get unreadClarifications() {
+    return (
+      this.activeTab !== 'clarifications' || this.clarificationsHasBeenRead
```

Comment:
no entiendo esto: si `this.clarificationsHaveBeenRead` es verdadero, regresamos que existen clarificaciones no-leídas?

no debería ser

```suggestion
      this.activeTab !== 'clarifications' && !this.clarificationsHaveBeenRead
```

? se puede agregar una prueba?

Reply:
Ok, la agrego.